internal/comment/github.go

```diff
@@ -0,0 +1,480 @@
+				createdAt:   createdAt.Time,
+				url:         string(node.URL),
+				isMinimized: bool(node.IsMinimized),
+			})
+		}
+		if !q.Repository.PullRequest.Comments.PageInfo.HasNextPage {
+			break
+		}
+		variables["after"] = githubv4.NewString(q.Repository.PullRequest.Comments.PageInfo.EndCursor)
+	}
+
+	var matchingComments []Comment
+	for _, comment := range allComments {
+		if strings.Contains(comment.Body(), tag) {
+			matchingComments = append(matchingComments, comment)
+		}
+	}
+
+	return matchingComments, nil
+}
+
+// CallCreateComment calls the GitHub API to create a new comment on the pull request.
+func (h *githubPRHandler) CallCreateComment(ctx context.Context, body string) (Comment, error) {
+	comment, _, err := h.v3client.Issues.CreateComment(
+		ctx,
+		h.owner,
+		h.repo,
+		h.prNumber,
+		&github.IssueComment{Body: github.String(body)},
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	return &githubComment{
+		globalID:    comment.GetNodeID(),
+		id:          int(comment.GetID()),
+		body:        comment.GetBody(),
+		createdAt:   comment.GetCreatedAt(),
+		url:         comment.GetHTMLURL(),
+		isMinimized: false,
+	}, nil
+}
+
+// CallUpdateComment calls the GitHub API to update the body of a comment on the pull request.
+func (h *githubPRHandler) CallUpdateComment(ctx context.Context, comment Comment, body string) error {
+	var m struct {
+		UpdateIssueComment struct {
+			ClientMutationId githubv4.ID //nolint
+		} `graphql:"updateIssueComment(input: $input)"`
+	}
+
+	input := githubv4.UpdateIssueCommentInput{
+		ID:   githubv4.NewString(githubv4.String(comment.(*githubComment).globalID)),
+		Body: githubv4.String(body),
+	}
+
+	return h.v4client.Mutate(ctx, &m, input, nil)
+}
+
+// CallDeleteComment calls the GitHub API to delete the pull request comment.
+func (h *githubPRHandler) CallDeleteComment(ctx context.Context, comment Comment) error {
+	var m struct {
+		DeleteIssueComment struct {
+			ClientMutationId githubv4.ID //nolint
+		} `graphql:"deleteIssueComment(input: $input)"`
+	}
+
+	input := githubv4.DeleteIssueCommentInput{
+		ID: githubv4.NewString(githubv4.String(comment.(*githubComment).globalID)),
+	}
+
+	return h.v4client.Mutate(ctx, &m, input, nil)
+}
+
+// CallHideComment calls the GitHub API to minimize the pull request comment.
+func (h *githubPRHandler) CallHideComment(ctx context.Context, comment Comment) error {
+	var m struct {
+		MinimizeComment struct {
+			ClientMutationId githubv4.ID //nolint
+		} `graphql:"minimizeComment(input: $input)"`
+	}
+
+	input := githubv4.MinimizeCommentInput{
+		SubjectID:  githubv4.NewString(githubv4.String(comment.(*githubComment).globalID)),
+		Classifier: githubv4.ReportedContentClassifiersOutdated,
+	}
+
+	return h.v4client.Mutate(ctx, &m, input, nil)
+}
+
+// githubCommitHandler is a PlatformHandler for GitHub commits. It
+// implements the PlatformHandler interface and contains the functions
+// for finding, creating, updating, deleting comments on GitHub commits.
```

Comment:
Should we mention this? :)

```suggestion
// for finding, creating, updating, deleting and hiding comments on GitHub commits.
```